horovod/mxnet/adapter.h

```diff
@@ -16,6 +16,8 @@
 #ifndef HOROVOD_MXNET_ADAPTER_H
 #define HOROVOD_MXNET_ADAPTER_H
 
+#include <mxnet/base.h>
+
```

Comment:
nit: remove blank line

Reply:
do we need to have a blank line between third party library and header files from itself?

Reply:
fine.